vtex-apps/b2b-organizations
# fix: skip graphql query if parameter is null

#### What problem is this solving?

We were observing the error `Variable "$email" of required type "String!" was not provided` on the logs and this PR avoid calls that were causing the error.
Now the `getBinding` query is skipped if the `userEmail` parameter is empty (resulting in the same behaviour)

#### How to test it?


Login and go to My Account (Profile) page
Looking at the Network tab of the browser inspector you can observe the error is not happening (compared to a workspace without the fix)
[Workspace](https://binding--b2bstoreqa.myvtex.com/account#/profile)
